bundles/org.openhab.binding.denonmarantz/src/main/java/org/openhab/binding/denonmarantz/internal/connector/telnet/DenonMarantzTelnetClientThread.java

```diff
@@ -65,7 +66,10 @@ public void run() {
 
             do {
                 try {
-                    String line = in.readLine();
+                    String line = null;
+                    if (in != null) {
+                        line = in.readLine();
```

Comment:
This seems to cause a new compiler warning, '''DenonMarantzTelnetClientThread.java:[71,32] Potential null pointer access: this expression has a '@Nullable' type'''.

Although there should be only one thread, you will probably need a local copy of '''in''' to get rid of this message.

Reply:
Fixed.